src/test/java/com/wikia/webdriver/common/clicktracking/ClickTrackingSupport.java

```diff
@@ -9,21 +9,17 @@
 
 public class ClickTrackingSupport {
 
-  public void compare(List<JsonObject> expectedEventList, List<JsonObject> currentEventList) {
+  public void compare(List<JsonObject> expectedEventList, List<JsonObject> trackedEventList) {
     for (JsonObject expectedEvent : expectedEventList) {
       Boolean equals = false;
```

Comment:
I do have to initialize this variable. Then it's value is determined within for loop, later used in if block, line 26.
This case is considered in java best practice article as permissible, read:
http://www.javapractices.com/topic/TopicAction.do?Id=126

thanks for the comment :+1: